data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -722,6 +722,30 @@
     ],
     "flags": [ "ALLOW_ROTTEN" ]
   },
+  {
+    "type": "recipe",
+    "result": "soap_flakes",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": "15 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "soap", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
```

Comment:
This recipe is probably unnecessary; don't think there's any item that provides CUT 2 quality.

Reply:
according to amurkin there is one

Reply:
There are `integrated toolset` and `extended toolset`.
Can they be used for crafting?

Reply:
Integrated toolset comes from having integrated toolset CBM installed; I think it was agreed that it really should never be better than standalone tools (except for "portability part), just like other multitools.